populate/src/main/java/bio/terra/pearl/populate/service/KitRequestPopulator.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.pearl.populate.service;
+
+import bio.terra.pearl.core.dao.kit.KitRequestDao;
+import bio.terra.pearl.core.dao.kit.KitTypeDao;
+import bio.terra.pearl.core.model.kit.KitRequest;
+import bio.terra.pearl.populate.dto.kit.KitRequestDto;
+import bio.terra.pearl.populate.service.contexts.FilePopulateContext;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Optional;
+
+// TODO: finish implementing this
```

Comment:
I've been winging it with these populators. They all seem a little bit different and the semantics of what each of the method implementions should do isn't 100% clear to me. Let me know if `KitTypePopulator` looks right and I'll come back and fill this in.